extensions/sql-database-projects/src/test/newProjectTool.test.ts

```diff
@@ -0,0 +1,71 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import * as should from 'should';
+import * as newProjectTool from '../tools/newProjectTool';
+import * as constants from '../common/constants';
+import { generateTestFolderPath, createTestFile } from './testUtils';
+
+describe.skip('NewProjectTool: New project tool tests', function (): void {
+
+	it('Should generate correct default project names', async function (): Promise<void> {
+		// save original setting
+		const previousSetting = await vscode.workspace.getConfiguration(constants.dbProjectConfigurationKey)[constants.projectSaveLocationKey];
+		const testFolderPath = await generateTestFolderPath();
+
+		try {
+			await vscode.workspace.getConfiguration(constants.dbProjectConfigurationKey).update(constants.projectSaveLocationKey, testFolderPath, true);
+			should(newProjectTool.defaultProjectNameNewProj()).equal('DatabaseProject1');
+			should(newProjectTool.defaultProjectNameFromDb('master')).equal('DatabaseProjectmaster');
+		}
+		finally {
+			// reset configuration
+			await vscode.workspace.getConfiguration(constants.dbProjectConfigurationKey).update(constants.projectSaveLocationKey, previousSetting, true);
```

Comment:
reseting configuration can be done in a beforeEach() or afterEach() so that it doesn't have to be done in every test